docs/user-guide/helm.md

```diff
@@ -1,87 +1,100 @@
-
-If you are using `Helm 2`, use the following command instead:
+2. If you are using Helm 3, install the Ambassador Edge Stack Chart with the following command:
 
 ```
-helm install --namespace ambassador --name ambassador stable/ambassador
+helm install ambassador --namespace ambassador datawire/ambassador
 ```
 
-2. Enable all Ambassador Edge Stack features with the following `kubectl` command:
+If you are using Helm 2, use the following command instead:
 
 ```
-kubectl apply -f https://www.getambassador.io/early-access/yaml/ambassador/ambassador-edge-stack-resources.yaml # FIXME: this URL is broken
+helm install --name ambassador --namespace ambassador datawire/ambassador
 ```
 
-The Helm chart deploys The Ambassador Edge Stack but does not create the Edge Stack resources necessary for authentication, rate limiting, automatic HTTPS, etc.
+This will install the necessary deployments, RBAC, Custom Resource Definitions, etc. for the Ambassador Edge Stack to route traffic. Details on how to configure Ambassador using the Helm chart can be found in the Helm chart [README](https://github.com/datawire/ambassador-chart/tree/master).
+
+**Note:** The Helm chart deploys the Ambassador Edge Stack, but it does not create the Edge Stack resources necessary for authentication, rate limiting, automatic HTTPS, etc.
 
-## Upgrade an Existing Installation
+## Upgrading an Existing Edge Stack Installation
 
-Upgrading your existing Ambassador Edge Stack is as simple as upgrading any other Helm release. Complete the following steps:
+**Note:** If your existing installation is not already running Ambassador **Edge Stack** as opposed to Ambassador Open Source, **do not use these instructions**. See "Migrating to Ambassador Edge Stack" below.
 
-1. Upgrade the release with the following command:
+Upgrading an existing installation of Ambassador Edge Stack is a two-step process:
+
+1. First, apply any CRD updates (as of Helm 3, this is not supported in the chart itself):
+
+```
+kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml
+```
+
+2. Next, upgrade the Ambassador Edge Stack itself:
 
 ```
-helm upgrade ambassador stable/ambassador
+helm upgrade ambassador datawire/ambassador
 ```
 
 This will upgrade the image and deploy and other necessary resources for the Ambassador Edge Stack. 
 
-**Note:** It is strongly encouraged for you to move your Ambassador release to the `ambassador` namespace with the following command: 
-`helm upgrade --namespace ambassador ambassador stable/ambassador`
+## Migrating to Ambassador Edge Stack
+
+If you have an existing Ambassador Open Source installation, but are not yet running Ambassador Edge Stack, the upgrade process is somewhat different than above.
```

Comment:
Ambassador API Gateway (we want to stay away from calling it the OSS bc it's confusing - richard's request)

Reply:
👍